pkgs/applications/version-management/git-and-tools/radicle-upstream/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, appimageTools, gsettings-desktop-schemas, gtk3, patchelf, zlib, glibc, fetchurl }:
+
+let
+  pname = "radicle-upstream";
+  version = "0.1.4";
+  name = "${pname}-${version}";
+in
+
+appimageTools.wrapAppImage {
+  inherit name;
+  extraPkgs = pkgs : [];
+
+  profile = ''
+    export XDG_DATA_DIRS=${gsettings-desktop-schemas}/share/gsettings-schemas/${gsettings-desktop-schemas.name}:${gtk3}/share/gsettings-schemas/${gtk3.name}:$XDG_DATA_DIRS
+  '';
+
+  extraInstallCommands = ''
+    mv $out/bin/${name} $out/bin/${pname}
```

Comment:
```suggestion
    mv $out/bin/radicle-upstream $out/bin/radicle-upstream
```

Reply:
I don't really understand the proposed change here?

Currently `appimageTools.wrapType2` will drop a binary with the name of the package (including the version number) in `$out/bin`, this line currently just renames that to remove the version number. 

Your proposed change would be a noop.